bundles/framework/divmanazer/component/Grid.js

```diff
@@ -14,6 +14,7 @@ Oskari.clazz.define('Oskari.userinterface.component.Grid',
         this.model = null;
         this._defaultLocKey = 'Grid';
         this._loc = this._getLocalization('DivManazer');
+        this.interpolate = Oskari.getMsg.bind(null, 'DivManazer');
```

Comment:
Does this really need both getLocalization and getMsg? Refactor Grid to use only getMsg()

Reply:
Both fetch the localization which is not done twice and kept in two separate variables.